docs/src/content/docs/guides/i18n.mdx

```diff
@@ -360,15 +360,15 @@ const allStrings = Astro.locals.t.all();
 
 #### `t.exists()`
 
-To check if a translation key exists for a locale, use the `locals.t.exists()` function with the translation key as first argument.
-Pass an optional second argument if you need to override the current locale.
+To check if a translation key exists, use the `locals.t.exists()` function with the translation key as first argument.
+Pass an optional second argument if you need to check if a key exists in a specific locale without considering any fallback logic.
```

Comment:
Maybe this might be a clearer way to say this?

```suggestion
Pass an optional second argument if you need to check if a translation exists for a specific locale.
```

“fallback logic” is kind of how we think about it internally, but I guess from a more user-facing POV, the thing you’re checking is if a _translation_ is available?

Reply:
Oh, I really like that, so much better 🧠 Thanks